boards/arm/nucleo_f091rc/pinmux.c

```diff
@@ -22,6 +22,16 @@ static const struct pin_config pinconf[] = {
 	{STM32_PIN_PA2, STM32F0_PINMUX_FUNC_PA2_USART2_TX},
 	{STM32_PIN_PA3, STM32F0_PINMUX_FUNC_PA3_USART2_RX},
 #endif	/* CONFIG_UART_STM32_PORT_2 */
+#ifdef CONFIG_SPI_1
```

Comment:
Please also update the board documentation to reflect that SPI is now a supported feature.

Maybe even add it to defconfig.

Reply:
I'll add it to the doc, I'm unsure about defconfig since there is multiple different pins for SPI1 & SPI2, and it depends on the usage